unix/vncserver/vncsession.c

```diff
@@ -134,7 +134,8 @@ begin_daemon(void)
 static void
 finish_daemon(void)
 {
-    write(daemon_pipe_fd, "+", 1);
+    if (write(daemon_pipe_fd, "+", 1) == -1)
+        fprintf(stderr, "Failed to write() to daemon_pipe_fd.\n");
```

Comment:
This is not going to work. stderr is set to /dev/null at this point.

Handling this error is probably not crucial. So might just want to silence the compiler warning.

Reply:
Done.
Seems like `if (...) {}` is the best way to silence this, as casting the result to `(void)...` no longer works.